bbdd/src/bbdd/Consultas.java

```diff
@@ -0,0 +1,119 @@
+package bbdd;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class Consultas {
+	
+	private java.sql.Connection conexionConn;
+	private final SentenciasBBDD sentenciasBBDD = new SentenciasBBDD();
+	static final String Transaccion="select max(Transaccion) from actividad";
+
+	public Consultas(Conexion conexion) {
+		this.conexionConn =  conexion.getConn();
+	}
+	
+	public ArrayList<String[]> conseguirLocales() {
```

Comment:
La base de datos no puede depender del modelo de una aplicación. Resolved esa dependencia.

Reply:
No me parece una buena idea, si pasamos un result set estamos obligados a mantener abierta la declaración y la conexión, Mantenerlos abiertos haría que estuvieran colgando y que la base de datos se quedara sin recursos cuando hay demasiados abiertos.
Si se cerrase la conexión cerraría implícitamente el conjunto de resultados.

¿Aún así procedemos?

Reply:
Si. De todas formas, si tenéis cualquier duda mañana me lo consultáis.